static/data/layers.js

```diff
@@ -0,0 +1,718 @@
+export default {
+    "version":8,
+    "metadata":{
+        "maputnik:renderer":"mbgljs"
+    },
+    "sources":{
+        "osmTiles":{
+            "type":"raster",
+            "tiles":[
+                "https://a.tile.openstreetmap.org/{z}/{x}/{y}.png",
+                "https://b.tile.openstreetmap.org/{z}/{x}/{y}.png",
+                "https://c.tile.openstreetmap.org/{z}/{x}/{y}.png"
+            ],
+            "tileSize":256,
+            "maxzoom":19,
+            "paint":{
+                "raster-fade-duration":100
+            },
+            "attribution":"data © <a target=\"_top\" rel=\"noopener\" href=\"https://www.openstreetmap.org/copyright\">OpenStreetMap</a> contributors."
+        },
+        "mediaWikiTiles":{
+            "type":"raster",
+            "tiles":[
+                "https://maps.wikimedia.org/osm-intl/{z}/{x}/{y}.png?lang=uk"
+            ],
+            "tileSize":256,
+            "maxzoom":19,
+            "paint":{
+                "raster-fade-duration":100
+            },
+            "attribution":"map: <a target=\"_top\" rel=\"noopener\" href=\"https://foundation.wikimedia.org/wiki/Maps_Terms_of_Use\">WikiMedia</a>; data: © <a target=\"_top\" rel=\"noopener\" href=\"https://www.openstreetmap.org/copyright\">OpenStreetMap</a> contributors."
+        },
+        "osmData":{
+            "type":"geojson",
+            "data":"https://dopomoha.pl/data/osm_data.geojson",
```

Comment:
Hey these links to the domain break local development; if we want to support a work flow where people use this file locally and work on the data download and processing steps, we should revert to serving this geojson file from the local server.

Reply:
Hi @daniel-j-h , yeah we made a decision to have it that way.

We wanted to separate map style from main JS file to be able to use Maputnik editor. With a visual editor we could have non-programmer folks edit the style. We decided that easier setup for the other folks would be more useful and for dev purposes one could temporarily edit he URL to point to local version of the file.
(there may be a better way to make both possible but that wasn't immediately clear)

Today we gathered enough frontend developers and they reached critical mass and decided to rewrite the site in react ;)

This is all very ~~chaotic~~ dynamic situation. If you want to talk on voice chat we're mostly communicating via discord at https://openstreetmap.org.pl/discord we could better keep in sync about the situation. My nick there is "tomczk".

We really appreciate all the enthusiasm and work on these Pull Requests that we received.

Going back to the satellite layer matter we encountered a problem with using the Mapbox Satellite layer and Maputnik editor and I'm not sure where lies the problem. The style works when loading the site locally but when we try to edit the style using Maputnik editor it just goes into endless loop and the map doesn't load at all. Not sure if we are doing something wrong or if it's a bug in the editor but it's kind of a problem since we really want to be able to use simple visual editor and Maputnik seemed perfect for that since it didn't require an account and allowed easy JSON access to be able to add custom metadata. Until we figure this out we'll have to hold off on adding the satellite layer to the map.